api_core/setup.py

```diff
@@ -12,68 +12,72 @@
+description = 'Google API client core library'
+version = '0.1.5.dev1'
+# Should be one of:
+# 'Development Status :: 3 - Alpha'
+# 'Development Status :: 4 - Beta'
+# 'Development Status :: 5 - Stable'
+release_status = 'Development Status :: 4 - Beta'
+dependencies = [
+    'googleapis-common-protos<2.0dev,>=1.5.3',
+    'protobuf>=3.0.0',
+    'google-auth<2.0.0dev,>=0.4.0',
+    'requests<3.0.0dev,>=2.18.0',
+    'setuptools>=34.0.0',
+    'six>=1.10.0',
+    'pytz',
+]
+extras = {
+    'grpc': 'grpcio>=1.8.2',
+    ':python_version < "3.2"': 'futures>=3.2.0',
+}
 
-with open(os.path.join(PACKAGE_ROOT, 'README.rst')) as file_obj:
-    README = file_obj.read()
 
+# Setup boilerplate below this line.
 
-SETUP_BASE = {
-    'author': 'Google Cloud Platform',
-    'author_email': 'googleapis-publisher@google.com',
-    'scripts': [],
-    'url': 'https://github.com/GoogleCloudPlatform/google-cloud-python',
-    'license': 'Apache 2.0',
-    'platforms': 'Posix; MacOS X; Windows',
-    'include_package_data': True,
-    'zip_safe': False,
-    'classifiers': [
-        'Development Status :: 4 - Beta',
+package_root = os.path.abspath(os.path.dirname(__file__))
+
+with io.open(os.path.join(package_root, 'README.rst')) as readme_file:
+    readme = readme_file.read()
+
+setuptools.setup(
+    name=name,
+    description=description,
+    version=version,
+    author='Google LLC',
+    author_email='googleapis-packages@google.com',
+    license='Apache 2.0',
+    classifiers=[
+        release_status,
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Apache Software License',
-        'Operating System :: OS Independent',
+        'Programming Language :: Python',
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Operating System :: OS Independent',
         'Topic :: Internet',
     ],
-}
-
-
-REQUIREMENTS = [
-    'googleapis-common-protos >= 1.5.3, < 2.0dev',
-    'protobuf >= 3.0.0',
-    'google-auth >= 0.4.0, < 2.0.0dev',
-    'requests >= 2.18.0, < 3.0.0dev',
-    'setuptools >= 34.0.0',
-    'six >= 1.10.0',
-    # pytz does not adhere to semver and uses a year.month based scheme.
-    # Any valid version of pytz should work for us.
-    'pytz',
-]
-
-EXTRAS_REQUIREMENTS = {
-    ':python_version<"3.2"': ['futures >= 3.2.0'],
-    'grpc': ['grpcio >= 1.8.2'],
-}
-
-setup(
-    name='google-api-core',
-    version='0.1.5.dev1',
-    description='Core Google API Client Library',
-    long_description=README,
+    platforms='Posix; MacOS X; Windows',
+    zip_safe=False,
+    include_package_data=True,
+    long_description=readme,
+    install_requires=dependencies,
+    extras_require=extras,
+    packages=setuptools.find_packages(exclude=('tests*',)),
```

Comment:
Might have to exclude "benchmark" folder too.https://github.com/GoogleCloudPlatform/google-cloud-python/tree/60d4460d13c2ce79ad085368f72cb93e97d7e6f9/bigquery/benchmark